app/Jobs/UpdateReply.php

```diff
@@ -12,18 +13,21 @@ final class UpdateReply
     private $reply;
 
     /**
-     * @var string
+     * @var array
      */
-    private $body;
+    private $attributes;
 
-    public function __construct(Reply $reply, string $body)
+    public function __construct(Reply $reply, UpdateReplyRequest $request)
```

Comment:
Can we just pass along the user object here of the one that edited the thread as `$updatedBy`? 

```suggestion
    public function __construct(Reply $reply, User $updatedBy, string $body)
```

Reply:
Done.